comptests/s3/test_rest_api_rgw.py

```diff
@@ -282,3 +284,339 @@ def test_user_create_36665(self):
+        status = loop.run_until_complete(self.obj.put_user_policy(user_params2))
+        self.log.info(
+            "Step 6: verifying Applied user policy to the tenant user")
+        assert status[0] == HTTPStatus.OK, "Not able to put user policy"
+        self.log.info(
+            "Step 7:  Applied user policy Successfully")
+        self.log.info(
+            "Step 8:  Deleting user policy for tenant user")
+        user_params3 = {
+            'UserName' : self.tenant +'$'+user_name,
+            'PolicyName' : 'Policy1',
+            'Action' : 'DeleteUserPolicy',
+            'format' : 'json'
+        }
+        loop = asyncio.get_event_loop()
+        status = loop.run_until_complete(self.obj.delete_user_policy(user_params3))
+        self.log.info(status)
+        assert status[0] == HTTPStatus.OK, "Not able to delete policy"
+        self.log.info(
+            "Step 9: Deleted Applied user policy to the tenant user")
+        self.created_users.append(user_params2)
+        self.log.info("END: %s",test_case_name)
+
+    @pytest.mark.api_user_ops
+    @pytest.mark.tags('TEST-41411')
+    def test_41411(self):
+        """Test to delete policy which does not exist"""
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Create IAM user")
+        user_name = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
+        email=f"{user_name}@seagate.com"
+        user_params = {
+            'display-name': user_name,
+            'email' : email,
+            'uid' : user_name
+        }
+        self.log.info("Step 2: Started creating IAM user.")
+        loop = asyncio.get_event_loop()
+        status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
+        self.log.info(user_info)
+        self.log.info(
+            "Step 3: Verifying that IAM user is created successfully")
+        assert status == HTTPStatus.OK, "Not able to create user. Test Failed"
+        self.log.info(
+            "Step 4: IAM user is created successfully")
+        self.log.info(
+            "Step 5:  Deleting user policy for with invalid policy name")
+        user_params2 = {
+            'UserName' : user_name,
+            'PolicyName' : 'Policy1',
+            'Action' : 'DeleteUserPolicy',
+            'format' : 'json'
+        }
+        self.log.info(
+            "Step 6:  Deleting user policy for with invalid policy name")
+        loop = asyncio.get_event_loop()
+        status = loop.run_until_complete(self.obj.delete_user_policy(user_params2))
+        assert status[0] == HTTPStatus.NOT_FOUND, "User policy deleted"
+        self.log.info("Step 7: Verified test to remove invalid policy name")
+        self.log.info("END : test to validate delete invalid policy")
+        self.created_users.append(user_params)
+        self.log.info("END: %s",test_case_name)
+    # pylint: disable=C0301
```

Comment:
Do not suppress [C0301 (line-too-long)](https://pylint.pycqa.org/en/latest/user_guide/messages/convention/line-too-long.html)

Reply:
Ok, got it will update requested changes